commands/highlight.js

```diff
@@ -0,0 +1,110 @@
+// Get the Highlights Table stored in the SQLite database
+const Highlights = require('../databaseFiles/highlightsTable.js');
+const Discord = require('discord.js');
+const config = require('../config.json');
+
+// Error handler
+const errHandler = err => {
+	console.error('Highlights sequelize error: ', err);
+};
+
+module.exports.execute = async (client, message, args) => {
+	const cmd = args[0]; // The command of what to do with the following phrase
+	const entirePhrase = args.join(' ');
+	const keywords = entirePhrase.substring(entirePhrase.indexOf(' ')+1); // Remove the first word, i.e. the command
+	const user = message.author;
+	const userID = user.id;
+	const sunEmote = config.emotes.highlights;
```

Comment:
Maybe update sunEmote to highlightsEmote? If it is changed to something else in the future, it'll be a funky name

Reply:
Sorry, that was incredibly lazy on my part.